classes/PodsAPI.php

```diff
@@ -8331,220 +8331,13 @@ public function process_form ( $params, $obj = null, $fields = null, $thank_you
 	 * Get current language information from Multilingual plugins
 	 *
 	 * @since 2.6.6
+	 * @since 2.7 Logic moved to PodsI18n
 	 *
 	 * @return array
 	 */
 	public static function get_current_language() {
```

Comment:
Oh, I remember. The reason I left it there is that it also calls `pods_i18n()` which includes the class if it doesn't exists yet.